mocca-client/src/test/java/com/paypal/mocca/client/MoccaClientQueryTest.java

```diff
@@ -177,6 +256,16 @@ public void moreThanOneMoccaAnnotationTest() {
         invalidClient.moreThanOneMoccaAnnotation("boo", "far");
     }
 
+    @Test
+    public void selectionSetCycleTest() {
+        try {
+            client.getResponseWithCycle();
+            fail("A Mocca exception was supposed to have been thrown!");
+        } catch (EncodeException e) {
+            assertEquals(e.getCause().getCause().getCause().getMessage(), "Selection set cannot be specified as there is a cycle in the return type caused by class com.paypal.mocca.client.sample.CyclePojo");
```

Comment:
Do we have any line wrapping conventions?  IMO 120 is max:)

Reply:
Done